### PR TITLE
MOO-646: Move the baseline sections to the Visit actions sections

### PR DESCRIPTION
**Pull request by ssmusoke**
- Set formAddMetadata="no" for Medical History, Patient Targets, Refer Patient and exit from NCD forms so that their UI metadata is not loaded
- Change the extension point for the forms in msfcore_forms_extension.json to visitActions from Overall Actions
- Delete the forms metadata from the database on shutdown

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/msfcore/activator/HtmlFormsInitializer.java [hunk 1]
```diff
@@ -93,7 +94,10 @@ public synchronized void started() {
      * Stop the initializer
      */
     public void stopped() {
-        //TODO: Perhaps disable the forms?
+        // Remove form metadata
+        log.warn("Removing form metadata");
+        AdministrationService as = Context.getAdministrationService();
+        as.executeSQL("delete from form_resource;", false);
```
Comment (kaweesi) [inline api/src/main/java/org/openmrs/module/msfcore/activator/HtmlFormsInitializer.java]: wait a minute, doesn't this delete all form resources? CC @ssmusoke, @Guimino , @lauravignoli i wouldn't like that at all, and i also think there are other places where forms are copied

Comment (kaweesi) [reply api/src/main/java/org/openmrs/module/msfcore/activator/HtmlFormsInitializer.java]: there are probably better ways to delete by the forms api than this raw deletion

Comment (lauravignoli) [reply api/src/main/java/org/openmrs/module/msfcore/activator/HtmlFormsInitializer.java]: Also, @ssmusoke how will this affect the possibility of making changes to the forms by updating metadata and publishing those changes to other instances (this is an important msf requirement)?

Comment (ssmusoke) [reply api/src/main/java/org/openmrs/module/msfcore/activator/HtmlFormsInitializer.java]: 1. Form_Resource contains the mappings to the user interface locations. 

2. My expectation is that the metadata will be deployed each time the instance starts so should be okay. However we need to add a test case to ensure that this happens as expected

**Pull request merged**